Mathlib/Analysis/CStarAlgebra/CStarMatrix.lean

```diff
@@ -41,7 +41,7 @@ def CStarMatrix (m : Type*) (n : Type*) (A : Type*) := Matrix m n A
 
 namespace CStarMatrix
 
-variable {m n A B R S : Type*}
+variable {m n R S A B : Type*}
```

Comment:
I am blind to why this style point (ordering R S A B rather than A B R S) is desirable? Is this documented somewhere?